manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -112,6 +112,12 @@ _internal_defaults_do_not_set:
   # Set this if you install ztunnel with a name different from the default.
   trustedZtunnelName: ""
 
+  # Ambient mode only.
+  # Set this to true if you want to deploy selectors in the mutating webhook
+  # to allow the injection of ambient init containers.
```

Comment:
+1 on default to false.

The comment should explain why someone wants to enable this injection.